src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerProcessor.cs

```diff
@@ -0,0 +1,200 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement.ObjectTypes.Server
+{
+     /// <summary>
+    /// Class to manage affinity for 64 processors in an independent manner for I/O as well as processors
+    /// </summary>
+    internal sealed class AffinityMngr
+    {
+        private const int MAX32CPU = 32;
+        private static string[] configFields = new string[]
```

Comment:
Are MAX32CPU and configFields here still needed? I'm not seeing any references to them in this file

Reply:
Looks like there a few 32 literals in the Affinity class that could use MAX32CPU